lib/elastic-search/requests.js

```diff
@@ -4,23 +4,35 @@ const logger = require('../logger')
 // takes array of JSON-ified EsBib records and indexes them to index specified in env vars
 // indexing creates or overwrites existing doc
 const writeRecords = async (records) => {
-  logger.debug('Writing batch of ' + records.length + 'resources', records)
+  logger.debug('Writing batch of ' + records.length + ' resources')
+
+  let resp
   try {
-    const resp = await _indexGeneric(process.env.ELASTIC_RESOURCES_INDEX_NAME, records, false)
-    logger.debug('resp: ', JSON.stringify(resp))
-    logger.debug('Index: ' + (resp.errors ? 'Error' : 'Success'))
-    return resp
+    resp = await _indexGeneric(process.env.ELASTIC_RESOURCES_INDEX_NAME, records, false)
   } catch (e) {
-    logger.error('writeRecords error: ', e)
+    throw new Error(`Error writing to ES: ${e}`)
+  }
+
+  if (resp.errors) {
+    const errors = resp.items?.map((item) => item.index?.error)
```

Comment:
can you put a comment here describing whats going on? looks like elastic search returns errors per item on the response?